src/Raven.Server/Web/Studio/StudioCollectionsHandler.cs

```diff
@@ -178,10 +177,18 @@ private void WriteDocument(BlittableJsonTextWriter writer, JsonOperationContext
             if (first == false)
                 writer.WriteComma();
 
+            var arrayStubsJson = new DynamicJsonValue();
+            foreach (var p in arraysStubs)
```

Comment:
If we are not expecting duplicates, then we can build DJV right-away. Instead of using Dictionaries, you can add values to DJV.

Reply:
ok